### DISCUSSION TITLE
why does it hide the imports?

### DISCUSSION THREAD

**Discussion by peteblank** [ecosse3/nvim#153] [Q&A]
Why does it hide the imports?

**Comment by ecosse3** [comment]
https://github.com/ecosse3/nvim/issues/152

**Comment by ecosse3** [comment]
The reason that Ecovim automatically hides import is to reduce the number of lines in already huge files that you work with in JS/TS. If you work with React.js you can have a lot of imports, and usually you don't carry about them when going to file. You can easily fold/unfold it.
